.github/ISSUE_TEMPLATE/content-review-request.md

```
@@ -0,0 +1,68 @@
+title: Content Review Request - [INSULT ID]
+labels: content review request
+assignees: Terry-BrooksJr
+
+---
+
+name: Report an Insult
+title: Report API Insult
+description: >-
+  This form is used to request a content review for one of the following
+  actions:
+  	•	Reclassification of an explicit determination
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 3

(MD010, no-hard-tabs)